### PR TITLE
Add concourse pipeline

### PR DESCRIPTION
**Pull request by thullSL**

### CODE REVIEW

File: concourse/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM mono
+
+COPY binaries/ service/ 
+
+EXPOSE 9000
+
+CMD ["mono", "service/Speakers.exe"]
```
Comment (csscottlogic) [inline concourse/Dockerfile]: Should this be Sessions.exe?

File: concourse/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      build: code
+  - put: cluster
+
+resources:
+- name: code
+  type: git
+  source:
+    uri: https://github.com/BristechSRM/api-docs.git
+
+- name: sessions-trigger
```
Comment (csscottlogic) [inline concourse/pipeline.yml]: Presumably we'll have to add triggers for other microservice repos as and when?

Comment (thullSL) [inline concourse/pipeline.yml]: Yeah I don't think they're ready with docs yet. But trivial to add

File: concourse/Dockerfile (hunk 1 above)
Comment (thullSL) [inline concourse/Dockerfile]: My thoughts are it should be neither. I have no clue what's going on here :worried: lemme look into this

**Comment by thullSL** [comment]
That docker file didn't belong at all, it was copied by mistake. I've deleted it

**Comment by csscottlogic** [comment]
:shipit:

**Pull request merged**